ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -432,6 +432,12 @@ objects:
         PXE_KERNEL_URL="$(jq -r '.baseURI + .images["kernel"].path' /var/lib/openshift-install/rhcos.json)"
         # 4.3 unified 'metal-bios' and 'metal-uefi' into just 'metal'
         PXE_OS_IMAGE_URL="$(jq -r '.baseURI + (.images["metal-bios"].path // .images["metal"].path)' /var/lib/openshift-install/rhcos.json)"
+        if [[ $PXE_KERNEL_URL =~ "live" ]]; then
+          PXE_KERNEL_ARGS=""
+        else
+          PXE_KERNEL_ARGS="coreos.inst.image_url=${PXE_OS_IMAGE_URL}"
+        fi
+        # 4.3 unified 'metal-bios' and 'metal-uefi' into just 'metal', unused in 4.6
```

Comment:
Comment seems misplaced, I think you mean to change the one above?  But not worth a respin.

Reply:
Thanks, I'll clean that up in https://github.com/openshift/release/pull/10121